clang/test/Sema/aarch64-neon-without-target-feature.cpp

```diff
@@ -0,0 +1,33 @@
+// RUN: %clang_cc1 -triple aarch64-none-linux-gnu -target-feature +dotprod  -target-feature +fullfp16 -target-feature +fp16fml -target-feature +i8mm -target-feature +bf16 -verify -emit-llvm -o - %s
+
+// This test is testing the diagnostics that Clang emits when compiling without '+neon'.
```

Comment:
do we have a positive Sema tests to show that the intrinsics are available when the neon feature is present?

Reply:
No. But I think this is already covered by CodeGen tests for individual intrinsics, which need to have +neon to work.